nrf_security/Kconfig.legacy

```diff
@@ -254,6 +254,15 @@ config OBERON_ONLY_ENABLED
 
 menu "Legacy mbed TLS crypto APIs"
 
+config MBEDTLS_MPI_WINDOW_SIZE
+	int
+	default 6
```

Comment:
Maybe we can try to enforce the 6 if CC3XX is enabled. Just to avoid misconfigurations. So something like range 6 6 if CC3XX_BACKEND